seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,9 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.40*\\s*\\/\\s*2\\.*0*/.test(code), 'message: You should use the <code>/</code> operator.');",
+        "assert(code.match(/quotient/g).length === 3, 'message: The quotient variable should only be assigned once.');"
```

Comment:
@erictleung I've confirmed with Berkeley that it's alright to delete the solution. This should be good to go now.